src/utilities/include/public.h

```diff
@@ -580,6 +503,21 @@ void Dump(const std::vector<T>& vec) {
   }
   std::cout << std::endl;
 }
+
+template <typename T>
+void Diff(const std::vector<T>& a, const std::vector<T>& b) {
```

Comment:
I added an extra debug helper to quickly see just which parts of two vectors are not identical.